### PR TITLE
Added getter for ObjMgr::_itemTemplateStoreFast

### PR DESCRIPTION
**Pull request by Kaev**
**Changes proposed:** Added getter for ObjMgr::_itemTemplateStoreFast

**Target branch(es):** Master

**Tests performed:** Builds, Tested

**Why:** This is needed if you want to add a new ItemTemplate outside of ObjMgr

### CODE REVIEW

File: src/game/Globals/ObjectMgr.h [hunk 1]
```diff
@@ -721,7 +721,8 @@ class ObjectMgr
         CreatureAddon const* GetCreatureTemplateAddon(uint32 entry);
         ItemTemplate const* GetItemTemplate(uint32 entry);
         ItemTemplateContainer const* GetItemTemplateStore() const { return &_itemTemplateStore; }
-
+        std::vector<ItemTemplate*> const* GetItemTemplateStoreFast() const { return &_itemTemplateStoreFast; }
+
```
Comment (ShinDarth) [inline src/game/Globals/ObjectMgr.h]: can you please remove those whitespaces?

Comment (Kaev) [reply src/game/Globals/ObjectMgr.h]: Which do you mean? I just took the line above it as a reference

Comment (Rochet2) [reply src/game/Globals/ObjectMgr.h]: @Kaev https://i.imgur.com/kAOGNT5.png trailing whitespace (whitespace at the end of line).

Comment (Kaev) [reply src/game/Globals/ObjectMgr.h]: already done :)

**Comment by ShinDarth** [comment]
thanks for your contribution!

**Pull request merged**